src/extension.ts

```diff
@@ -556,22 +556,21 @@ function exposeKubernetes() {
         cmd += ' --port=' + ports[0];
     }
 
-    kubectl.invoke(cmd);
+    kubectl.invokeInTerminal(cmd);
```

Comment:
Expose should not go via terminal, per Kylie's notes.

Reply:
Fixed. Already undo the changes.